### ISSUE TITLE
Cron: Autoload path not allowed

### ISSUE THREAD

**Issue by devz3r0** [owncloud/core#23077]
### Steps to reproduce
1. Open Owncloud
2. browse around
3. check logfile

### Expected behaviour
no errors shoukd be there

### Actual behaviour
```json
{"reqId":"vcZkGlP0cVhSvhlES5A4","remoteAddr":"80.109.255.5","app":"core","message":"Error while running background job (class: OC\\Command\\CommandJob, arguments: O:33:\"OCA\\Files_Versions\\Command\\Expire\":4:{s:43:\"\u0000OCA\\Files_Versions\\Command\\Expire\u0000fileName\";s:75:\"Dokumente\/zim-home-crypt\/databases\/database-foYyApmfPRpVdoduyssX-0000000036\";s:47:\"\u0000OCA\\Files_Versions\\Command\\Expire\u0000versionsSize\";N;s:46:\"\u0000OCA\\Files_Versions\\Command\\Expire\u0000neededSpace\";i:0;s:39:\"\u0000OCA\\Files_Versions\\Command\\Expire\u0000user\";s:7:\"devzero\";}): {\"Exception\":\"OCP\\\\AutoloadNotAllowedException\",\"Message\":\"Autoload path not allowed: \\\/var\\\/www\\\/owncloud\\\/apps\\\/files_versions\\\/command\\\/expire.php\",\"Code\":0,\"Trace\":\"#0 \\\/var\\\/www\\\/owncloud\\\/lib\\\/autoloader.php(158): OC\\\\Autoloader->isValidPath('\\\/var\\\/www\\\/ownclo...')\\n#1 [internal function]: OC\\\\Autoloader->load('OCA\\\\Files_Versi...')\\n#2 [internal function]: spl_autoload_call('OCA\\\\Files_Versi...')\\n#3 \\\/var\\\/www\\\/owncloud\\\/lib\\\/private\\\/command\\\/commandjob.php(32): unserialize('O:33:\\\"OCA\\\\Files...')\\n#4 \\\/var\\\/www\\\/owncloud\\\/lib\\\/private\\\/backgroundjob\\\/job.php(52): OC\\\\Command\\\\CommandJob->run('O:33:\\\"OCA\\\\Files...')\\n#5 \\\/var\\\/www\\\/owncloud\\\/lib\\\/private\\\/backgroundjob\\\/queuedjob.php(42): OC\\\\BackgroundJob\\\\Job->execute(Object(OC\\\\BackgroundJob\\\\JobList), Object(OC\\\\Log))\\n#6 \\\/var\\\/www\\\/owncloud\\\/cron.php(163): OC\\\\BackgroundJob\\\\QueuedJob->execute(Object(OC\\\\BackgroundJob\\\\JobList), Object(OC\\\\Log))\\n#7 {main}\",\"File\":\"\\\/var\\\/www\\\/owncloud\\\/lib\\\/autoloader.php\",\"Line\":138}","level":3,"time":"2016-03-10T10:38:12+00:00"}
```

### Server configuration
**Operating system**:
Linux, Ubuntu 14.04
**Web server:**
Apache 2.4
**Database:**
mysql
**PHP version:**
5.5.9
**ownCloud version:** (see ownCloud admin page)
8.2.3
**Updated from an older ownCloud or fresh install:**
Updated from 8.2.2
**Where did you install ownCloud from:**
tar.bz2 File from owncloud.org

**List of activated apps:**
```
Enabled:
  - activity: 2.1.4
  - calendar: 0.8.2
  - contacts: 0.5.0.0
  - documents: 0.11.1
  - files: 1.2.1
  - files_external: 0.3.0
  - files_pdfviewer: 0.7
  - files_sharing: 0.7.0
  - files_videoviewer: 0.1.3
  - gallery: 14.2.0
  - ownnote: 1.05
Disabled:
  - encryption
  - external
  - files_texteditor
  - files_trashbin
  - files_versions
  - firstrunwizard
  - notifications
  - provisioning_api
  - templateeditor
  - updater
  - user_external
  - user_ldap
```

**The content of config/config.php:**
```php
<?php
$CONFIG = array (
  'installed' => true,
  'dbtype' => 'mysql',
  'dbname' => '****',
  'dbuser' => '****',
  'dbpassword' => '*****',
  'dbhost' => '127.0.0.1',
  'dbtableprefix' => 'oc_',
  'passwordsalt' => '****',
  'forcessl' => false,
  'blacklisted_files' => 
  array (
    0 => '.htaccess',
  ),
  'enable_previews' => true,
  'preview_max_scale_factor' => 10,
  'preview_libreoffice_path' => '//usr/bin/libreoffice',
  'defaultapp' => 'files',
  'knowledgebaseenabled' => true,
  'appstoreenabled' => true,
  'mail_smtpdebug' => false,
  'mail_smtpmode' => 'sendmail',
  'mail_smtphost' => '127.0.0.1',
  'mail_smtpport' => 25,
  'mail_smtptimeout' => 10,
  'mail_smtpauth' => false,
  'mail_smtpauthtype' => 'LOGIN',
  'trashbin_retention_obligation' => 7,
  'allow_user_to_change_display_name' => true,
  'updatechecker' => true,
  'log_type' => 'owncloud',
  'loglevel' => '3',
  'memcache.local' => '\\OC\\Memcache\\APCu',
  'remember_login_cookie_lifetime' => 1296000,
  'maintenance' => false,
  'instanceid' => '517fb9460260f',
  'datadirectory' => '/var/www/owncloud/data',
  'version' => '8.2.3.2',
  'trusted_domains' => 
  array (
    0 => '*****',
    1 => '******',
    2 => '****',
  ),
  'secret' => '******',
  'appstore.experimental.enabled' => true,
  'theme' => '',
);
```

**Are you using external storage, if yes which one:** local/smb/sftp/...
SMB, other ownclouds from different peoples,
**Are you using encryption:** yes/no
no (use a truecryptcontainer)
**Are you using an external user-backend, if yes which one:** LDAP/ActiveDirectory/Webdav/...

### Client configuration
**Browser:**
Firefox 45
**Operating system:**
Ubuntu 15.10
#### Web server error log
```
Insert your webserver log here
```

#### ownCloud log (data/owncloud.log)
```json
{"reqId":"OL6BsGZbog2wxJVmBn0s","remoteAddr":"......","app":"core","message":"Error while running background job (class: OC\\Command\\CommandJob, arguments: O:33:\"OCA\\Files_Versions\\Command\\Expire\":4:{s:43:\"\u0000OCA\\Files_Versions\\Command\\Expire\u0000fileName\";s:75:\"Dokumente\/zim-home-crypt\/databases\/database-foYyApmfPRpVdoduyssX-0000000035\";s:47:\"\u0000OCA\\Files_Versions\\Command\\Expire\u0000versionsSize\";N;s:46:\"\u0000OCA\\Files_Versions\\Command\\Expire\u0000neededSpace\";i:0;s:39:\"\u0000OCA\\Files_Versions\\Command\\Expire\u0000user\";s:7:\"devzero\";}): {\"Exception\":\"OCP\\\\AutoloadNotAllowedException\",\"Message\":\"Autoload path not allowed: \\\/var\\\/www\\\/owncloud\\\/apps\\\/files_versions\\\/command\\\/expire.php\",\"Code\":0,\"Trace\":\"#0 \\\/var\\\/www\\\/owncloud\\\/lib\\\/autoloader.php(158): OC\\\\Autoloader->isValidPath('\\\/var\\\/www\\\/ownclo...')\\n#1 [internal function]: OC\\\\Autoloader->load('OCA\\\\Files_Versi...')\\n#2 [internal function]: spl_autoload_call('OCA\\\\Files_Versi...')\\n#3 \\\/var\\\/www\\\/owncloud\\\/lib\\\/private\\\/command\\\/commandjob.php(32): unserialize('O:33:\\\"OCA\\\\Files...')\\n#4 \\\/var\\\/www\\\/owncloud\\\/lib\\\/private\\\/backgroundjob\\\/job.php(52): OC\\\\Command\\\\CommandJob->run('O:33:\\\"OCA\\\\Files...')\\n#5 \\\/var\\\/www\\\/owncloud\\\/lib\\\/private\\\/backgroundjob\\\/queuedjob.php(42): OC\\\\BackgroundJob\\\\Job->execute(Object(OC\\\\BackgroundJob\\\\JobList), Object(OC\\\\Log))\\n#6 \\\/var\\\/www\\\/owncloud\\\/cron.php(163): OC\\\\BackgroundJob\\\\QueuedJob->execute(Object(OC\\\\BackgroundJob\\\\JobList), Object(OC\\\\Log))\\n#7 {main}\",\"File\":\"\\\/var\\\/www\\\/owncloud\\\/lib\\\/autoloader.php\",\"Line\":138}","level":3,"time":"2016-03-10T10:28:57+00:00"}  
{"reqId":"R+0zF8+LhYyAojgMND8F","remoteAddr":".....","app":"core","message":"Error while running background job (class: OC\\Command\\CommandJob, arguments: O:33:\"OCA\\Files_Versions\\Command\\Expire\":4:{s:43:\"\u0000OCA\\Files_Versions\\Command\\Expire\u0000fileName\";s:88:\"Dokumente\/zim-home-crypt\/multichunks\/multichunk-60a56d3995a30337b75f0648d2dd361ea4b9a518\";s:47:\"\u0000OCA\\Files_Versions\\Command\\Expire\u0000versionsSize\";N;s:46:\"\u0000OCA\\Files_Versions\\Command\\Expire\u0000need
edSpace\";i:0;s:39:\"\u0000OCA\\Files_Versions\\Command\\Expire\u0000user\";s:7:\"devzero\";}): {\"Exception\":\"OCP\\\\AutoloadNotAllowedException\",\"Message\":\"Autoload path not allowed: \\\/var\\\/www\\\/owncloud\\\/apps\\\/files_versions\\\/command\\\/expire.php\",\"Code\":0,\"Trace\":\"#0 \\\/var\\\/www\\\/owncloud\\\/lib\\\/autoloader.php(158): OC\\\\Autoloader->isValidPath('\\\/var\\\/www\\\/ownclo...')\\n#1 [internal function]: OC\\\\Autoloader->load('OCA\\\\Files_Versi...')\\n#2 [internal function]: spl_autoload_call('OCA\\\\Files_Versi...')\\n#3 \\\/var\\\/www\\\/owncloud\\\/lib\\\/private\\\/command\\\/commandjob.php(32): unserialize('O:33:\\\"OCA\\\\Files...')\\n#4 \\\/var\\\/www\\\/owncloud\\\/lib\\\/private\\\/backgroundjob\\\/job.php(52): OC\\\\Command\\\\CommandJob->run('O:33:\\\"OCA\\\\Files...')\\n#5 \\\/var\\\/www\\\/owncloud\\\/lib\\\/private\\\/backgroundjob\\\/queuedjob.php(42): OC\\\\BackgroundJob\\\\Job->execute(Object(OC\\\\BackgroundJob\\\\JobList), Object(OC\\\\Log))\\n#6 \\\/var\\\/www\\\/owncloud\\\/cron.php(163): OC\\\\BackgroundJob\\\\QueuedJob->execute(Object(OC\\\\BackgroundJob\\\\JobList), Object(OC\\\\Log))\\n#7 {main}\",\"File\":\"\\\/var\\\/www\\\/owncloud\\\/lib\\\/autoloader.php\",\"Line\":138}","level":3,"time":"2016-03-10T10:32:05+00:00"}
{"reqId":"vcZkGlP0cVhSvhlES5A4","remoteAddr":"......","app":"core","message":"Error while runnin
g background job (class: OC\\Command\\CommandJob, arguments: O:33:\"OCA\\Files_Versions\\Command\\Expi
re\":4:{s:43:\"\u0000OCA\\Files_Versions\\Command\\Expire\u0000fileName\";s:75:\"Dokumente\/zim-home-c
rypt\/databases\/database-foYyApmfPRpVdoduyssX-0000000036\";s:47:\"\u0000OCA\\Files_Versions\\Command\
\Expire\u0000versionsSize\";N;s:46:\"\u0000OCA\\Files_Versions\\Command\\Expire\u0000neededSpace\";i:0
;s:39:\"\u0000OCA\\Files_Versions\\Command\\Expire\u0000user\";s:7:\"devzero\";}): {\"Exception\":\"OC
P\\\\AutoloadNotAllowedException\",\"Message\":\"Autoload path not allowed: \\\/var\\\/www\\\/owncloud
\\\/apps\\\/files_versions\\\/command\\\/expire.php\",\"Code\":0,\"Trace\":\"#0 \\\/var\\\/www\\\/ownc
loud\\\/lib\\\/autoloader.php(158): OC\\\\Autoloader->isValidPath('\\\/var\\\/www\\\/ownclo...')\\n#1 
[internal function]: OC\\\\Autoloader->load('OCA\\\\Files_Versi...')\\n#2 [internal function]: spl_aut
oload_call('OCA\\\\Files_Versi...')\\n#3 \\\/var\\\/www\\\/owncloud\\\/lib\\\/private\\\/command\\\/co
mmandjob.php(32): unserialize('O:33:\\\"OCA\\\\Files...')\\n#4 \\\/var\\\/www\\\/owncloud\\\/lib\\\/pr
ivate\\\/backgroundjob\\\/job.php(52): OC\\\\Command\\\\CommandJob->run('O:33:\\\"OCA\\\\Files...')\\n
#5 \\\/var\\\/www\\\/owncloud\\\/lib\\\/private\\\/backgroundjob\\\/queuedjob.php(42): OC\\\\Backgroun
dJob\\\\Job->execute(Object(OC\\\\BackgroundJob\\\\JobList), Object(OC\\\\Log))\\n#6 \\\/var\\\/www\\\
/owncloud\\\/cron.php(163): OC\\\\BackgroundJob\\\\QueuedJob->execute(Object(OC\\\\BackgroundJob\\\\Jo
bList), Object(OC\\\\Log))\\n#7 {main}\",\"File\":\"\\\/var\\\/www\\\/owncloud\\\/lib\\\/autoloader.ph
p\",\"Line\":138}","level":3,"time":"2016-03-10T10:38:12+00:00"}
{"reqId":"1gZMHCzj6FviVB5QKQk8","remoteAddr":".......","app":"core","message":"Error while runnin
g background job (class: OC\\Command\\CommandJob, arguments: O:33:\"OCA\\Files_Versions\\Command\\Expire\":4:{s:43:\"\u0000OCA\\Files_Versions\\Command\\Expire\u0000fileName\";s:88:\"Dokumente\/zim-home-crypt\/multichunks\/multichunk-2bb43b9832393a22d9c45c239daef0e1c40bdcaf\";s:47:\"\u0000OCA\\Files_Versions\\Command\\Expire\u0000versionsSize\";N;s:46:\"\u0000OCA\\Files_Versions\\Command\\Expire\u0000neededSpace\";i:0;s:39:\"\u0000OCA\\Files_Versions\\Command\\Expire\u0000user\";s:7:\"devzero\";}): {\"Exception\":\"OCP\\\\AutoloadNotAllowedException\",\"Message\":\"Autoload path not allowed: \\\/var\\\/www\\\/owncloud\\\/apps\\\/files_versions\\\/command\\\/expire.php\",\"Code\":0,\"Trace\":\"#0 \\\/var\\\/www\\\/owncloud\\\/lib\\\/autoloader.php(158): OC\\\\Autoloader->isValidPath('\\\/var\\\/www\\\/ownclo...')\\n#1 [internal function]: OC\\\\Autoloader->load('OCA\\\\Files_Versi...')\\n#2 [internal function]: spl_autoload_call('OCA\\\\Files_Versi...')\\n#3 \\\/var\\\/www\\\/owncloud\\\/lib\\\/private\\\/command\\\/commandjob.php(32): unserialize('O:33:\\\"OCA\\\\Files...')\\n#4 \\\/var\\\/www\\\/owncloud\\\/lib\\\/private\\\/backgroundjob\\\/job.php(52): OC\\\\Command\\\\CommandJob->run('O:33:\\\"OCA\\\\Files...')\\n#5 \\\/var\\\/www\\\/owncloud\\\/lib\\\/private\\\/backgroundjob\\\/queuedjob.php(42): OC\\\\BackgroundJob\\\\Job->execute(Object(OC\\\\BackgroundJob\\\\JobList), Object(OC\\\\Log))\\n#6 \\\/var\\\/www\\\/owncloud\\\/cron.php(163): OC\\\\BackgroundJob\\\\QueuedJob->execute(Object(OC\\\\BackgroundJob\\\\JobList), Object(OC\\\\Log))\\n#7 {main}\",\"File\":\"\\\/var\\\/www\\\/owncloud\\\/lib\\\/autoloader.php\",\"Line\":138}","level":3,"time":"2016-03-10T10:45:48+00:00"}
{"reqId":"LKBtj6AleA\/2hmrHWs2B","remoteAddr":".....","app":"core","message":"Error while running background job (class: OC\\Command\\CommandJob, arguments: O:33:\"OCA\\Files_Versions\\Command\\Expire\":4:{s:43:\"\u0000OCA\\Files_Versions\\Command\\Expire\u0000fileName\";s:75:\"Dokumente\/zim-home-crypt\/databases\/database-foYyApmfPRpVdoduyssX-0000000037\";s:47:\"\u0000OCA\\Files_Versions\\Command\\Expire\u0000versionsSize\";N;s:46:\"\u0000OCA\\Files_Versions\\Command\\Expire\u0000neededSpace\";i:0;s:39:\"\u0000OCA\\Files_Versions\\Command\\Expire\u0000user\";s:7:\"devzero\";}): {\"Exception\":\"OCP\\\\AutoloadNotAllowedException\",\"Message\":\"Autoload path not allowed: \\\/var\\\/www\\\/owncloud\\\/apps\\\/files_versions\\\/command\\\/expire.php\",\"Code\":0,\"Trace\":\"#0 \\\/var\\\/www\\\/owncloud\\\/lib\\\/autoloader.php(158): OC\\\\Autoloader->isValidPath('\\\/var\\\/www\\\/ownclo...')\\n#1 [internal function]: OC\\\\Autoloader->load('OCA\\\\Files_Versi...')\\n#2 [internal function]: spl_autoload_call('OCA\\\\Files_Versi...')\\n#3 \\\/var\\\/www\\\/owncloud\\\/lib\\\/private\\\/command\\\/commandjob.php(32): unserialize('O:33:\\\"OCA\\\\Files...')\\n#4 \\\/var\\\/www\\\/owncloud\\\/lib\\\/private\\\/backgroundjob\\\/job.php(52): OC\\\\Command\\\\CommandJob->run('O:33:\\\"OCA\\\\Files...')\\n#5 \\\/var\\\/www\\\/owncloud\\\/lib\\\/private\\\/backgroundjob\\\/queuedjob.php(42): OC\\\\BackgroundJob\\\\Job->execute(Object(OC\\\\BackgroundJob\\\\JobList), Object(OC\\\\Log))\\n#6 \\\/var\\\/www\\\/owncloud\\\/cron.php(163): OC\\\\BackgroundJob\\\\QueuedJob->execute(Object(OC\\\\BackgroundJob\\\\JobList), Object(OC\\\\Log))\\n#7 {main}\",\"File\":\"\\\/var\\\/www\\\/owncloud\\\/lib\\\/autoloader.php\",\"Line\":138}","level":3,"time":"2016-03-10T10:46:14+00:00"}
```

#### Browser log
```
Content Security Policy: Directive 'frame-src' has been deprecated. Please use directive 'child-src' instead. <unknown>
Content Security Policy: The page's settings blocked the loading of a resource at self ("script-src https://...... 'unsafe-eval'"). files:1:0
Using //@ to indicate sourceMappingURL pragmas is deprecated. Use //# instead jquery.min.js:1:0
Content Security Policy: The page's settings blocked the loading of a resource at self ("script-src https://...... 'unsafe-eval'"). files
window.controllers is deprecated. Do not use it for UA detection. nsHeaderInfo.js:412:8
Use of getPreventDefault() is deprecated.  Use defaultPrevented instead.
```

**Comment by DeepDiver1975** [comment]
Looks like files_version is disabled but there are still jobs in the queue

**Comment by devz3r0** [comment]
i disabled that ages ago, that error just arised now, how can i clear that ajax-jobs out?

**Comment by nickvergessen** [comment]
Delete entry manually from oc_jobs database table for the time being.

**Comment by devz3r0** [comment]
in that table there are 3529 entries. Can I just wipe it entirely?

**Comment by nickvergessen** [comment]
No, try:

```sql
DELETE FROM `oc_jobs`
WHERE `class` LIKE '%Files\_Versions%'
```

**Comment by RealRancor** [comment]
Just an additional info. The user had initially reported this at the forums: https://forum.owncloud.org/viewtopic.php?f=36&t=34210

There he provided the initial reason for that messges. It seems a symlink was used:

> in /var/www/owncloud/ that is linked from
owncloud -> /media/owncloud/
thats from:
dev/mapper/truecrypt1 on /media/owncloud type ext4 (rw)

https://forum.owncloud.org/viewtopic.php?f=36&t=34210#p110023

**Comment by devz3r0** [comment]
that query didn't work because the entries are as followed:

[code]
7 	OC\BackgroundJob\Legacy\RegularJob 	["OCA\\Music\\Backgroundjob\\CleanUp","run"] 	1442406780
8 	OCA\Files_sharing\Lib\DeleteOrphanedSharesJob 	null 	1442408549
9 	OCA\Activity\BackgroundJob\EmailNotification 	null 	1442465408
10 	OCA\Activity\BackgroundJob\ExpireActivities 	null 	1442403427
585 	OC\Command\CommandJob 	"O:33:\"OCA\\Files_Versions\\Command\\Expire\":4:{s:43:\"\u0000OCA\\Files_Versions\\Command\\Expire\u0000fileName\";s:97:\"Dokumente\/zim-home-crypt\/temporary\/temp-IzTSF-multichunk-c635d3a4fa56c5ed50e7f50cd019f2b264eab845\";s:47:\"\u0000OCA\\Files_Versions\\Command\\Expire\u0000versionsSize\";N;s:46:\"\u0000OCA\\Files_Versions\\Command\\Expire\u0000neededSpace\";i:0;s:39:\"\u0000OCA\\Files_Versions\\Command\\Expire\u0000user\";s:7:\"devzero\";}" 	0
586 	OC\Command\CommandJob 	"O:33:\"OCA\\Files_Versions\\Command\\Expire\":4:{s:43:\"\u0000OCA\\Files_Versions\\Command\\Expire\u0000fileName\";s:97:\"Dokumente\/zim-home-crypt\/temporary\/temp-sNtOu-multichunk-3785ea2dd7af2186dbf384c61873be2cb10b2fe3\";s:47:\"\u0000OCA\\Files_Versions\\Command\\Expire\u0000versionsSize\";N;s:46:\"\u0000OCA\\Files_Versions\\Command\\Expire\u0000neededSpace\";i:0;s:39:\"\u0000OCA\\Files_Versions\\Command\\Expire\u0000user\";s:7:\"devzero\";}" 	0
587 	OC\Command\CommandJob 	"O:33:\"OCA\\Files_Versions\\Command\\Expire\":4:{s:43:\"\u0000OCA\\Files_Versions\\Command\\Expire\u0000fileName\";s:97:\"Dokumente\/zim-home-crypt\/temporary\/temp-PMECa-multichunk-60a56d3995a30337b75f0648d2dd361ea4b9a518\";s:47:\"\u0000OCA\\Files_Versions\\Command\\Expire\u0000versionsSize\";N;s:46:\"\u0000OCA\\Files_Versions\\Command\\Expire\u0000neededSpace\";i:0;s:39:\"\u0000OCA\\Files_Versions\\Command\\Expire\u0000user\";s:7:\"devzero\";}" 	0
588 	OC\Command\CommandJob 	"O:33:\"OCA\\Files_Versions\\Command\\Expire\":4:{s:43:\"\u0000OCA\\Files_Versions\\Command\\Expire\u0000fileName\";s:97:\"Dokumente\/zim-home-crypt\/temporary\/temp-Sgwvx-multichunk-73d38c4db698ac42ba8fb4942a62c6088465a0c0\";s:47:\"\u0000OCA\\Files_Versions\\Command\\Expire\u0000versionsSize\";N;s:46:\"\u0000OCA\\Files_Versions\\Command\\Expire\u0000neededSpace\";i:0;s:39:\"\u0000OCA\\Files_Versions\\Command\\Expire\u0000user\";s:7:\"devzero\";}" 	0
589 	OC\Command\CommandJob 	"O:33:\"OCA\\Files_Versions\\Command\\Expire\":4:{s:43:\"\u0000OCA\\Files_Versions\\Command\\Expire\u0000fileName\";s:97:\"Dokumente\/zim-home-crypt\/temporary\/temp-HQIBW-multichunk-6fd6ce028f4ca2eaa394832ef81df1ea51e455a0\";s:47:\"\u0000OCA\\Files_Versions\\Command\\Expire\u0000versionsSize\";N;s:46:\"\u0000OCA\\Files_Versions\\Command\\Expire\u0000neededSpace\";i:0;s:39:\"\u0000OCA\\Files_Versions\\Command\\Expire\u0000user\";s:7:\"devzero\";}" 	0
590 	OC\Command\CommandJob 	"O:33:\"OCA\\Files_Versions\\Command\\Expire\":4:{s:43:\"\u0000OCA\\Files_Versions\\Command\\Expire\u0000fileName\";s:97:\"Dokumente\/zim-home-crypt\/temporary\/temp-kePOh-multichunk-a1f7a0f7bd5465a2d5489704f0c8a0d507a7f084\";s:47:\"\u0000OCA\\Files_Versions\\Command\\Expire\u0000versionsSize\";N;s:46:\"\u0000OCA\\Files_Versions\\Command\\Expire\u0000neededSpace\";i:0;s:39:\"\u0000OCA\\Files_Versions\\Command\\Expire\u0000user\";s:7:\"devzero\";}" 	0
591 	OC\Command\CommandJob 	"O:33:\"OCA\\Files_Versions\\Command\\Expire\":4:{s:43:\"\u0000OCA\\Files_Versions\\Command\\Expire\u0000fileName\";s:97:\"Dokumente\/zim-home-crypt\/temporary\/temp-fzvKb-multichunk-a1e993f8652be4f75c53488430e73880d807d417\";s:47:\"\u0000OCA\\Files_Versions\\Command\\Expire\u0000versionsSize\";N;s:46:\"\u0000OCA\\Files_Versions\\Command\\Expire\u0000neededSpace\";i:0;s:39:\"\u0000OCA\\Files_Versions\\Command\\Expire\u0000user\";s:7:\"devzero\";}" 	0
[/code]

till to:
[code]
4106 	OC\Command\CommandJob 	"O:33:\"OCA\\Files_Versions\\Command\\Expire\":4:{s:43:\"\u0000OCA\\Files_Versions\\Command\\Expire\u0000fileName\";s:75:\"Dokumente\/zim-home-crypt\/databases\/database-foYyApmfPRpVdoduyssX-0000000181\";s:47:\"\u0000OCA\\Files_Versions\\Command\\Expire\u0000versionsSize\";N;s:46:\"\u0000OCA\\Files_Versions\\Command\\Expire\u0000neededSpace\";i:0;s:39:\"\u0000OCA\\Files_Versions\\Command\\Expire\u0000user\";s:7:\"devzero\";}" 	0
4107 	OC\Command\CommandJob 	"O:33:\"OCA\\Files_Versions\\Command\\Expire\":4:{s:43:\"\u0000OCA\\Files_Versions\\Command\\Expire\u0000fileName\";s:75:\"Dokumente\/zim-home-crypt\/databases\/database-gDdgWcJYrQkPkjlKZvFo-0000000100\";s:47:\"\u0000OCA\\Files_Versions\\Command\\Expire\u0000versionsSize\";N;s:46:\"\u0000OCA\\Files_Versions\\Command\\Expire\u0000neededSpace\";i:0;s:39:\"\u0000OCA\\Files_Versions\\Command\\Expire\u0000user\";s:7:\"devzero\";}" 	0
4108 	OC\Command\CommandJob 	"O:33:\"OCA\\Files_Versions\\Command\\Expire\":4:{s:43:\"\u0000OCA\\Files_Versions\\Command\\Expire\u0000fileName\";s:45:\"Dokumente\/zim-home-crypt\/databases\/cleanup-53\";s:47:\"\u0000OCA\\Files_Versions\\Command\\Expire\u0000versionsSize\";N;s:46:\"\u0000OCA\\Files_Versions\\Command\\Expire\u0000neededSpace\";i:0;s:39:\"\u0000OCA\\Files_Versions\\Command\\Expire\u0000user\";s:7:\"devzero\";}" 	0
4109 	OCA\Files\BackgroundJob\CleanupFileLocks 	null 	0
[/code]

**Comment by nickvergessen** [comment]
Right, 8.2 I forgot. Then try:
```sql
DELETE FROM `oc_jobs`
WHERE `class` LIKE '%CommandJob' AND `argument` LIKE 'O:33%Files_Versions%'
```

**Comment by devz3r0** [comment]
that worked, thanks there are now only the 5 entries left.
It seems to be quiet now.
as I don't use files_versions there is still a folder in data/devzero/files_versions
I have the question: can I delete that, or does it break something?

**Comment by nickvergessen** [comment]
I can't tell you something about that.

**Comment by PVince81** [comment]
@devz3r0 if you disable the files_version app you can delete these folders. You should then also to delete the matching entries in "oc_filecache" table.

@nickvergessen what's the next step ? Is there any cron class name cleanup code that needs to be implemented ?

**Comment by nickvergessen** [comment]
Well on 9.0 the jobs are removed because the class does not exist anymore. So if we want to fix something, could do a cleanup in 8.2, but that is kind of odd, sine it only happens when the app is disabled, so it can not be part of the app's update.

So I'd say nothing to do here for us.

**Comment by PVince81** [comment]
Ok then, closing
